plugins/node/opentelemetry-instrumentation-connect/src/instrumentation.ts

```diff
@@ -52,7 +52,7 @@ export class ConnectInstrumentation extends InstrumentationBase<Server> {
 
   init() {
     return [
-      new InstrumentationNodeModuleDefinition<any>(
```

Comment:
This is already typed as `any` so the `moduleExports` type is not typed at the moment. Can be improved in a future PR. want to keep this one minimal to the relevant changes